site/content/docs/include/common-svc-fields.ja.md

```diff
@@ -34,8 +34,22 @@ command: ["ps", "au"]
 
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
+<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
+`docker build --platform` で渡す、デフォルト以外のオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
+
+自動生成された文字列を上書きして、異なる `osfamily` や `architecture` でビルドをします。例えば、Windows ユーザーは次の文字列を
+
+```yaml
+platform: windows/amd64
+```
+
+Map を使って、デフォルトの `WINDOWS_SERVER_2019_CORE` から変更する可能性があります。
```

Comment:
Removing this line would be nice as I included this part in the above paragraph. What do you think?

Reply:
Yes, I think it is very good. I will create a new commit.